boggs/src/main/java/com/hartwig/healthchecks/boggs/extractor/BoggsExtractor.java

```diff
@@ -1,35 +1,48 @@
 package com.hartwig.healthchecks.boggs.extractor;
 
-import java.io.FileNotFoundException;
+import static java.util.stream.Collectors.toCollection;
+import static java.util.stream.Collectors.toList;
+
+import java.io.File;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Wrong order for 'java.io.File' import.](https://www.codacy.com/app/hmf/health-checks/file/2780681836/issues/source?bid=3363458&fileBranchId=3365368#l6)